open_samus_returns_rando/samus_returns_patcher.py

```diff
@@ -48,6 +65,8 @@ def _wrap(v: str):
         ),
         "starting_scenario": _wrap(starting_location["scenario"]),
         "starting_actor": _wrap(starting_location["actor"]),
+        "energy_per_tank": energy_per_tank,
+        "aeion_per_tank": aeion_per_tank,
```

Comment:
Is this for future™ because there is nothing in `custom_init.lua` for `energy_per_tank` and `aeion_per_tank`?

Reply:
Uh no I forgot to add it